staging/github.com/seal-io/utils/stringx/string.go

```diff
@@ -89,3 +89,12 @@ func ReplaceFunc(s string, rp func(rune) rune) string {
 	}
 	return b.String()
 }
+
+// LastContent retrieves the last characters of a string.
+func LastContent(content string, length int) string {
+	if len(content) < length {
+		return content
+	}
+
+	return content[len(content)-length:]
+}
```

Comment:
LastContent causes one memory assigned.

for Golang, getting the last content of a string is very easy to achieve.

let's remove this function, and use string[last_content_start:].